src/integration.spec.ts

```diff
@@ -0,0 +1,84 @@
+import { WorkOS } from './workos';
+
+function random(length: number) {
+  var result = '';
+  var characters =
+    'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+  var charactersLength = characters.length;
+  for (var i = 0; i < length; i++) {
+    result += characters.charAt(Math.floor(Math.random() * charactersLength));
+  }
+  return result;
+}
+
+describe('WorkOS', () => {
```

Comment:
will you also try a couple list organization memberships error cases? userId doesn't exist / organizationId doesn't exist?

otherwise this looks great!